docs/install.md

```diff
@@ -176,42 +175,7 @@ To do this, you first need to:
 -   Apply for a Season of your new Show (List My Shows -> New Season)
 -   Schedule the Season (Shows Scheduler)
 
-### Setting up your own 2016-site
```

Comment:
is there a PR to add these to the 2016-site repo?

Reply:
PR has been added 
https://github.com/UniversityRadioYork/2016-site/pull/372
Also is this what you meant by the comment above or did you mean something else I missed

Reply:
nope, the comment is about something different (i linked to a specific comment)